sdk/python/feast/errors.py

```diff
@@ -85,10 +85,11 @@ def __init__(self, online_store_class_name: str):
         )
 
 
-class FeastOnlineStoreConfigInvalidName(Exception):
-    def __init__(self, online_store_config_class_name: str):
+class FeastStoreConfigInvalidName(Exception):
+    def __init__(self, online_store_config_class_name: str, store_type="Online"):
         super().__init__(
-            f"Online Store Config Class '{online_store_config_class_name}' should end with the string `OnlineStoreConfig`.'"
+            f"Online Store Config Class '{online_store_config_class_name}' "
+            f"should end with the string `{store_type}Config`.'"
```

Comment:
It's not clear what `OnlineConfig` here means. Shouldn't this be the name of the Config class? (assuming the default value is used)

Reply:
Sorry, I don't know what you're referring to here - the diff may have changed since the last time you looked at it